### DISCUSSION TITLE
Stuck on Windows logo, latest Ubuntu on Surface Laptop 3

### DISCUSSION THREAD

**Discussion by Mountanalog** [linux-surface/linux-surface#1385] [Q&A] [answered]
So, I had no business attempting this as I have next to no knowledge or experience with anything like this. But I did it and now I'm stuck with an unusable laptop. Just wanted to get my Surface Laptop 3 off of windows. Installed Ubuntu 22.04.4 LTS from a USB stick and followed step by step guide to install surface kernel and upon reboot couldn't get past the windows logo. I don't have another laptop. Unsure what to do. I have searched the forums and found similar problems but none of the solutions worked for me or maybe I didn't understand them. Any help is appreciated.

**Comment by Mountanalog** [comment]
I have tried different combinations of boot order. I don't have a way to recover to windows. Is there a way to reinstall Ubuntu from the USB and just start over that way? I can't get to the install Ubuntu screen

**Reply by galbers** [reply, marked as answer]
I had a somewhat similar issue. I wasn't able to boot back into the Ubuntu installer after the initial run.

I had to fully disable all secure boot features of the laptop, and when I installed Ubuntu the second time I did NOT do the linux-surface-secureboot-mok steps.  After all that everything works fine for me.

To reinstall Ubuntu a second time I had to use Ventoy.  The laptop was able to boot off of the flash drive with Ventoy on it and I was able to get everything working at that point.
https://www.ventoy.net/en/doc_start.html
